src/Utilities/Compiler/DisposeMethodKind.cs

```diff
@@ -5,7 +5,7 @@ namespace Analyzer.Utilities
     /// <summary>
     /// Describes different kinds of Dispose-like methods.
     /// </summary>
-    public enum DisposeMethodKind
+    internal enum DisposeMethodKind
```

Comment:
Please don't do this, the public APIs from these package are consumed externally in Roslyn repo.

Reply:
OH! I'll have to work harder on it, then, as there are conflicts because it's public in more than one assembly.

Reply:
Removing reference from project MetaCompilation.Analyzers to shared project Analyzer.Utilities solved the conflicts.